worker/src/main/java/gov/cms/ab2d/worker/config/AutoScalingServiceImpl.java

```diff
@@ -111,6 +129,16 @@ private void scaleBackToMin() {
         }
     }
 
+    @Override
+    public void onApplicationEvent(PropertiesChangedEvent propertiesChangedEvent) {
+        corePoolSize = Integer.parseInt(propertiesChangedEvent.getPropertiesMap().get(PCP_CORE_POOL_SIZE).toString());
+        maxPoolSize = Integer.parseInt(propertiesChangedEvent.getPropertiesMap().get(PCP_MAX_POOL_SIZE).toString());
+        scaleToMaxTime = Double.parseDouble(propertiesChangedEvent.getPropertiesMap().get(PCP_SCALE_TO_MAX_TIME).toString());
+
+        this.executor.setCorePoolSize(corePoolSize);
+        this.executor.setMaxPoolSize(maxPoolSize);
```

Comment:
No need to modify max pool size on executor, since the Auto Scaling service dynamically manages that value.